tests/test_editable.py

```diff
@@ -189,3 +190,39 @@ def test_editble_reentrant(venv, editable_imports_itself_during_build):
         assert venv.python('-c', 'import plat; print(plat.data())').strip() == 'DEF'
     finally:
         path.write_text(code)
+
+
+def test_editable_verbose(venv, editable_complex, monkeypatch):
+    monkeypatch.setenv('MESONPY_EDITABLE_VERBOSE', '1')
+    venv.pip('install', os.fspath(editable_complex))
+
+    # First import should have no output since wheel has already been built
+    assert venv.python('-c', 'import complex').strip() == ''
+
+    # Add empty line a pyx, make sure that the Compiling lines are seen
+    complex_package_dir = venv.python(
+        '-c', 'import os; import complex; print(os.path.dirname(complex.__file__))').strip()
+    cython_path = pathlib.Path(complex_package_dir).parent / 'test.pyx'
+    cython_content = cython_path.read_text()
+    try:
+        cython_path.write_text(cython_content + '\n')
+        output = venv.python('-c', 'import complex').strip()
+        # Need to filter some warning on OSX like
+        # 'ld: warning: -undefined dynamic_lookup may not work with chained fixups'
+        output_lines = [line for line in output.splitlines() if 'warning' not in line]
+        expected_pattern_list =  [
+            'meson-python: building complex',
+            'meson-python build command:.+(ninja|samu)',
+            'Compiling Cython source',
+            'Compiling C object',
+            'Linking target'
+        ]
```

Comment:
Why do you check for the exact content of the log messages? It seems fragile (they are not under the control of meson-python) and not really required. The output is also very different on Windows.

Reply:
I was a bit overoptimistic indeed, I am going to only check the first two lines of output since they are under meson-python control.